docker/test/libfuzzer/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG FROM_TAG=latest
+FROM clickhouse/test-base:$FROM_TAG
+
+# ARG for quick switch to a given ubuntu mirror
+ARG apt_archive="http://archive.ubuntu.com"
+RUN sed -i "s|http://archive.ubuntu.com|$apt_archive|g" /etc/apt/sources.list
+
+ENV LANG=C.UTF-8
+ENV TZ=Europe/Amsterdam
```

Comment:
Is it so necessary? What about randomizing?

Reply:
well... it's a (not exact) copy-paste from `docker/test/fuzzer/Dockerfile` - I think it's meant to be static when not intended specifically for testing, and for testing of these parameters we will need dedicated tests anyway